UpdateManagerの追加

KeigoNakamura opened the pull request:
UpDateManagerの追加
・GameManager.csの追加
・各スクリプトのUpdateで呼ばれる物の追加
・Shot内にInputの追加
inputManagerの更新
・代替キーの設定

NaitoKeisuke committed:
ひとまず変更点をコミット。
ボタンに対応はひとまず待機

KeigoNakamura committed:
inputManagerの変更を更新
・特定のキーをキーボード側から入力できるようにしました

KeigoNakamura committed:
UpDateManagerの追加

KeigoNakamura committed:
Merge pull request #27 from YutaKaseda/debug

debugをUpDateManagerにマージ

KeigoNakamura committed:
debug との競合

KeigoNakamura committed:
UpDateManagerの更新
・各スクリプトのUpdateで呼ばれるものの追加
・Shot内にinputの追加

YutaKaseda commented on OculusGame/Assets/Player2D/Bullet.cs:
```diff
@@ -9,8 +9,9 @@ public class Bullet : MonoBehaviour {
-	void ShotMove(){
+
+    public void ShotMove()
+    {
```
なんでここpublicにしたの？

YutaKaseda commented on OculusGame/Assets/Manager/GameManager.cs:
```diff
@@ -0,0 +1,33 @@
+	// Update is called once per frame
+	void Update () {
+		player3dMove.GetComponent<Player3dMove> ().Player3DMove ();
+		player3dMove.GetComponent<Player3DBulletShot> ().ShotBullet ();
```
なんで弾の発射までUpdateで呼んだの？
Moveとかの中で弾が発射されたらbulletprefabが呼ばれて、アタッチされたスクリプトの中のAwake()が呼ばれて発射されるようになってるはず

YutaKaseda commented on OculusGame/Assets/Player2D/Bullet.cs:
```diff
@@ -9,8 +9,9 @@ public class Bullet : MonoBehaviour {
-	
-	void ShotMove(){
+
+    public void ShotMove()
```
ここをpublicにする必要なし。
Awake()でもう呼ばれてます。

YutaKaseda commented on OculusGame/Assets/Player3D/Script/BulletStandard.cs:
```diff
@@ -15,7 +15,8 @@ public class BulletStandard : MonoBehaviour {
 
 	//弾が前に動く
-	void ShotBullet(){
+    public void ShotBullet()
```
ここもpublicいらず

YutaKaseda commented on OculusGame/Assets/Player3D/Script/Player3DBulletShot.cs:
```diff
@@ -10,13 +10,10 @@ public class Player3DBulletShot : MonoBehaviour {
 	//弾を発射（弾を呼び出す）
-	void ShotBullet(){
-		Instantiate(player3DBullet,transform.position,transform.rotation);
+	public void ShotBullet(){
```
これもかな？

YutaKaseda commented on OculusGame/Assets/Manager/GameManager.cs:
```diff
@@ -0,0 +1,33 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class GameManager : MonoBehaviour {
```
これの役割はGameのマネージメントじゃないので、class名は不適切。
名前をObjectUpdaterとかUpdaterにかえてください

KeigoNakamura committed:
UpDateManagerの更新
指摘箇所の改善をしました
・GameManager.csの名前を変更しました
・Bullet.cs Player2D.cs BulletStandard.cs Player3dMove.cs の一部を改変しました
・PLayer3DBulletShot.csを削除しました

KeigoNakamura committed:
・TagManager.asset　の追加
・Collision.mata　の改良
